core/mapper/exprmapper/expression/direction/direction.go

```diff
@@ -107,9 +115,12 @@ func NewArgument(a Attribute) (interface{}, error) {
 	case *token.Token:
 		param.Type = funcexprtype.STRING
 		param.Value = string(a.(*token.Token).Lit)
-	case int64:
+	case int:
 		param.Type = funcexprtype.INTEGER
 		param.Value = a
+	case float64:
```

Comment:
would there ever be the possibility of a float32 coming in?

Reply:
Hi @mellistibco  so far no.   the expr/function parser only support int and float64 for now.